kakao/KakaoLoginServer/app.js

```diff
@@ -126,7 +200,7 @@ app.post('/verifyToken', (req, res) => {
   createFirebaseToken(token).then((firebaseToken) => {
     console.log(`Returning firebase token to user: ${firebaseToken}`);
     res.send({firebase_token: firebaseToken});
-  });
+  }).catch((error) => res.status(401).send({message: error}));
```

Comment:
maybe I would change `message` to `error` since it's really an error more than a message :)

Important point: in the Android Client application we need to display a special message when `error.code === 'auth/existing-account-need-auth';` and in that case we need to link the account if the user signs-in with the existing account. To do that we need to add a new endpoints to just "link accounts" where you would pass the `kakaoAccessToken` and the existing firebase user's ID Token.